Section_prices/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Section Prices</title>
+    <link <link
+        href="https://fonts.googleapis.com/css?family=Poppins:400,600|Roboto:300,400,700&display=swap&subset=latin-ext"
+        rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="pricesWrap">
+        <main>
+            <div class="choosePlan">
+                <h1>Choose your <span class="blue">plan</span></h1>
+                <p>Now for manners use has company believe parlors. Least nor party who wrote while did. Excuse formed
+                    as is agreed admire so on result parish. Put use set uncommonly announcing and travelling. Allowance
+                    sweetness direction to as necessary.</p>
+            </div>
+
+            <section class="container">
+                <div class="price">
+                    <div class="logo"></div>
+
+                    <p class="priceBold">Basic</p>
+                    <p class="priceMoney">Free</p>
+                    <p>1 Project</p>
+                    <p>1 Customer</p>
+                    <p>4 Custom Tags</p>
+                    <p>Up to 5 people in the team</p>
+                    <p>123</p>
+                    <p>456</p>
+                    <p>789</p>
+                    <button class="priceStart">Start Today</button>
+                </div>
+                <div class="price">
+                    <div class="logo"></div>
```

Comment:
Wiem, że w Vue każdy komponent to oddzielne style, ale co jeżeli kiedyś zachcę stworzyć globalny styl dla klasy logo? Proszę o nazywanie klas w sposób bardziej szczegółowy. Nie ma problemu aby nazwać klase "priceLogo"